TrackletAlgorithm/InputRouter.h

```diff
@@ -226,8 +226,9 @@ void InputRouter( const BXType bx
 	
 	#pragma HLS inline
 	#pragma HLS interface register port = bx_o
-  
-  	ap_uint<1> hIs2S= hLinkWord.range(kLINKMAPwidth-4,kLINKMAPwidth-4);
+	#pragma HLS array_partition variable = hOutputStubs complete
```

Comment:
@sseifeln are you happy with these InputRouter changes?

Reply:
Yes happy with those!